### ISSUE TITLE
Command `drush en` doesn't work for submodules

### ISSUE THREAD

**Issue by jenlampton** [backdrop-contrib/drush#108]
I can't seem to enable the `flexslider_views` module using `drush en`. The main Flexslider module enables okay, but for the submodule I get the error:
```
Error flexslider_views does not exist in your Backdrop installation.
Try downloading flexslider_views first with the command: drush dl flexslider_views
```

**Comment by serundeputy** [comment]
Thanks Jen!

Need to make the module search recursive through sub directories.
This is dependent on #7.

**Comment by serundeputy** [comment]
I've updated the way modules are detected by drush in #7 and in my tests that fixes this as well i.e. I can enable `flexslider_views` with drush on the branch `ongoing-saga-of-en`

**Comment by jenlampton** [comment]
I'm not sure your changes made into the latest code for #7. I'm still unable to enable submodules:
```bash
> drush en devel_generate -y

	Error devel_generate does not exist in your Backdrop installation.
	Try downloading devel_generate first with the command: drush dl devel_generate
```

**Comment by serundeputy** [comment]
this was fixed in #7 
closing.
